prac_09/taxi_test.py

```diff
@@ -0,0 +1,18 @@
+# Adrianto Hartono
+# 14347930
+
+from prac_09.taxi import Taxi
+
+def main():
+    my_taxi = Taxi("Prius 1", 100)
```

Comment:
I think you should add 1.23

Reply:
Thank you for the feedback, but the 1.23 already imported from prac_09.taxi , hopefully it helps to answer your doubt🙏